style.css

```diff
@@ -1,39 +1,119 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter');
-body{
-    box-sizing: border-box;
+@import url("https://fonts.googleapis.com/css2?family=cyrillic Roboto");
+@import url("https://fonts.googleapis.com/css2?family=Poppins");
+@import url("https://fonts.googleapis.com/css2?family=Crete Round");
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  width: 375px;
+  height: 100vh;
+  margin: 0 auto;
+}
+
+h1 {
+  font-weight: 100;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.header {
+  background: #3c3a39;
+  height: 50px;
+  width: 375px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-p {
+  margin-left: 15px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
 }
-ul li{
-    list-style-type: none;
-    margin-right: 15px;
+
+.fa-bars-icon {
+  margin-right: 15px;
+  font-size: 18px;
+  color: #fff;
 }
-a{
-    text-decoration: none;
-    font-family: Inter;
-    color: white;
+
+.section-1 {
+  background-size: 406px 311px;
+  height: 590px;
+  width: 375px;
+  background-color: black;
+  background-image: url("./images/background-image.svg");
+  background-repeat: no-repeat;
+  background-position-x: 116px;
+  background-position-y: 310px;
 }
-main{
-    background: #1C1A19;
 
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  height: 590px;
+  width: 375px;
 }
-header{
-    background: #3E3C3C;
-    color: white;
-    display: flex;
-    padding-left: 30px;
+
+.primary-text {
+  width: 327px;
+  height: 124px;
+  font-family: "Crete Round", cursive;
+  font-style: normal;
+  font-size: 36px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-left: 20px;
+  margin-bottom: 40px;
 }
-.header-nav-div{
-    margin-left: 30%;
+
+.supporting-text {
+  width: 327px;
+  height: 27px;
+  margin-left: 20px;
+  margin-bottom: 60px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff5e1;
 }
-.header-name-div p{
-    font-family: Inter;
-    color: white;
+
+.description {
+  width: 327px;
+  height: 123px;
+  margin-left: 20px;
+  margin-bottom: 10px;
+  font-family: "Poppins", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social-media {
+  margin-left: -33px;
+}
+
+.social-media ul {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  justify-content: flex-start;
 }
-.fa-envelope{
-    font-family: Inter;
-    color: white;
-    margin: auto 30px;
 
+.social-media ul li {
+  margin-right: 10px;
```

Comment:
- [ ] I believe this was the illustration image the previous review was talking about, but I see the design issue was not fixed yet as you can see down below, consider fixing it and resubmit your work.

<img width="423" alt="1" src="https://user-images.githubusercontent.com/77843050/212488338-2c939ecb-0838-4431-9cf3-4c0a30b6319f.png">

Reply:
@joskalenda 
 About the illustration image positioning, I don't know why on my side it shows it has been displayed well.
![Screenshot 2023-01-15 135447](https://user-images.githubusercontent.com/103370560/212536859-ae9192ef-ad9e-4f2a-9aca-12323e1da14f.png)

Reply:
Hmmm. So, it's actually a matter of different screen sizes. I think You have done your work, as of now. But you must keep in mind that your UI should be satisfying for various screen sizes.

Reply:
@RaoAkif 
  I have currently made changes concerning the responsiveness. Could you check if I have done the correct thing